indexer/src/main/java/au/org/aodn/esindexer/utils/DeliveryModeUtils.java

```diff
@@ -0,0 +1,80 @@
+package au.org.aodn.esindexer.utils;
+
+import au.org.aodn.metadata.iso19115_3_2018.MDMetadataType;
+
+import java.util.Optional;
+
+public class DeliveryModeUtils {
+    public enum DeliveryMode {
+        other("other"),
+        completed("completed"),
+        real_time("real-time"),
+        delayed("delayed");
+
+        private final String mode;
+
+        DeliveryMode(String mode) {
+            this.mode = mode;
+        }
+
+        @Override
+        public String toString() {
+            return mode;
+        }
+    }
+
+    protected static Optional<DeliveryMode> scanRealTimeOrDelayed(String text) {
+        if (text != null) {
+            text = text.toLowerCase();
+            if(text.contains("real time") || text.contains("realtime")) {
+                return Optional.of(DeliveryMode.real_time);
+            }
+            else if(text.contains("delayed")) {
```

Comment:
also incorrect spelling e.g `delay` or `delayedd` ?

Reply:
it is delayed